integrations/ipfy.go

```diff
@@ -0,0 +1,66 @@
+package integrations
+
+import (
+	"github.com/buger/jsonparser"
+	"io/ioutil"
+	"net"
+	"net/http"
+)
+
+const IpFyExtIPEndpoint = "https://api.ipify.org?format=json"
+
+
+type IPFy struct {
+	Endpoint string
+	Resolved net.IP
+}
+
+var ipFyApi *IPFy
+
+//InitIPFy
+//function to initialize IPFyApi
+func InitIPFy() (err error) {
+	ipFyApi,err = NewIPFy()
+	return
+}
+
+func NewIPFy()(*IPFy,error){
```

Comment:
Codacy found an issue: [exported function NewIPFy should have comment or be unexported](https://app.codacy.com/gh/go-go-dojo/friendly-potato/pullRequest?prid=6165294)